Dockerfile

```diff
@@ -1,35 +1,22 @@
-#   Setup a Python virtual environment
-FROM python:3.9.13-slim-bullseye AS python-builder
+# Setup a Python virtual environment
+FROM python:3.9-bullseye
```

Comment:
Why don't you use the `slim` version? Or it is the same?

Reply:
Because it does not feature CURL which we need to fetch the transifex client.

Reply:
What is the different size between the two? Isn't it cheaper to just `apt install curl`?

Reply:
Just checked, the difference is 350MB vs 50MB.

Reply:
Okay, thank you!